src/main/java/frc/trigon/robot/RobotContainer.java

```diff
@@ -48,21 +56,39 @@ private void initComponents() {
 
     private void initCommands() {
         swerveCmd = new FieldRelativeSupplierDrive(
-                () -> -controller.getLeftY(),
-                () -> controller.getLeftX(),
+                () -> controller.getLeftY(),
+                () -> -controller.getLeftX(),
                 () -> -controller.getRightX()
         );
         collectCommand = new CollectCommand();
 
         countBallsCommand = new CountBallsCommand();
         shotsDetectorCommand = new ShotsDetectorCommand();
+
+        turnToHubCmd = new TurnToTargetCommand(limelight, 0);
+
+        primeShooterCmd = Shooter.getInstance().getPrimeShooterCommandWithDefault(
+                () -> limelight.hasTarget() ?
+                      ShootingCalculations.getShootingVelocityFromDistance(limelight.getDistanceFromHub()) :
+                      0
```

Comment:
Not sure its the best idea to set to 0 if there is no target. i really see that causing issues

Reply:
What else?